### PR TITLE
Adding batch-wise member display on the About page

### PR DESCRIPTION
**Pull request by TheUltimateKeval**
## TODO
- [x] Implement collapsible sections to show members of a batch
- [x] Add batch tag to individual profile page
- [ ] Open form to register people from the 2023 batch

## ⚠️ NOTE ⚠️
This PR is in development stage and uses a copy of the original AirTable.
This is a reminder to myself and the repository owners to revert back to using the original table before merging the PR.

### CODE REVIEW

**Comment by TheUltimateKeval** [comment]
Collapsible sections:

image

image

**Comment by TheUltimateKeval** [comment]
Batch Tag:

image

**Comment by mrigankpawagi** [comment]
Looks good. 
1. Should we keep the latest batch's section open by default?
2. Maybe add caret icons to indicate that the sections wrap/unwrap. 
3. We can use the same Airtable and the same form eventually, with a new "batch" field.

**Comment by TheUltimateKeval** [comment]
> Looks good.
> 
>     1. Should we keep the latest batch's section open by default?
> 
>     2. Maybe add caret icons to indicate that the sections wrap/unwrap.
> 
>     3. We can use the same Airtable and the same form eventually, with a new "batch" field.

1. Yes I was thinking of doing the same.

2. I thought just changing the color to indicate collapsed / open sections would work but yeah adding caret icons will make it much more intuitive. Will get to that.

3. Yes that is the plan. I am only using a copy of the original AirTable for testing before we actually get people to submit forms for their profiles.

**Comment by mrigankpawagi** [comment]
More padding is needed here: 
image

I guess increase both top and bottom padding.

**Comment by mrigankpawagi** [comment]
For now, we can make all sections open by default.

**Comment by TheUltimateKeval** [comment]
> More padding is needed here:
> I guess increase both top and bottom padding.

I see the issue and increasing the padding as you suggested is one solution.

But the way current hover effect moves the text is quite disruptive to the UI. I think it should generally be avoided. Instead I am thinking of keeping the text static and shrinking the photo instead for the hover effect.

out

**Comment by mrigankpawagi** [comment]
Oh great, this is actually better!

**Comment by mrigankpawagi** [comment]
I had a hard time deciding the hover effect when I was making it for the first time. This is definitely cleaner.

**Comment by TheUltimateKeval** [comment]
Great! Now we are only waiting for the green flag for registering new members on the website. Any plans on when we are doing that?

**Comment by mrigankpawagi** [comment]
As we discussed: we use the same form with a new field (Batch) in it.

**Review by mrigankpawagi — APPROVED** [review]

**Comment by mrigankpawagi** [comment]
We'll merge once you revert to the original database.

**Review by sharaths21312 — COMMENTED** [review]
Looks good to me, although maybe the font size of the about paragraph could be increased slightly, I feel like the margins are a bit too small too.

**Comment by TheUltimateKeval** [comment]
> Looks good to me, although maybe the font size of the about paragraph could be increased slightly, I feel like the margins are a bit too small too.

The about paragraph itself needs updating considering the CSA Chair has changed. Should I just change the designation to **(Former Chair, CSA)** or are there other changes to be done?

Also can you elaborate about which margins are you talking about?

**Comment by sharaths21312** [comment]
Yes, changing the designation to "former" would be fine here.

I'm talking about the side margins on the text part, on a small screen/when
zoomed in it ends up extremely close to the edge of the window.

File: about/index.html [hunk 1]
```diff
@@ -108,7 +108,7 @@ <h1>About</h1>
           <br>
           <p> 
             Databased is headed by Masavir Khaliq, an M.Tech. student at CSA, IISc, and is run by students
-            of the B.Tech. (Maths and Computing) program. We are fortunate to have Chiranjib Bhattacharya (Chair, CSA) as our supervisor.
+            of the B.Tech. (Maths and Computing) program. We are fortunate to have Chiranjib Bhattacharya (Former Chair, CSA) as our supervisor.
```
Comment (mrigankpawagi) [inline about/index.html:111]: Ah yes! Let us hope we also get in touch with Prof. Vinod :)

**Review by mrigankpawagi — APPROVED** [review]

**Comment by mrigankpawagi** [comment]
Ready to merge?

File: about/index.html (hunk 1 above)
Comment (TheUltimateKeval) [reply about/index.html:111]: We certainly will in UMC102!

**Pull request merged**